.github/workflows/rust.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     needs: Formatting
     runs-on: ubuntu-latest
     env:
-      MSRV_VERSION: 1.65.0
+      MSRV_VERSION: 1.70.0
```

Comment:
It this bump needed? Do you use a feature of rust 1.70 that 1.65 does not yet have?